web/samples_index/pubspec.yaml

```diff
@@ -9,7 +9,8 @@ dependencies:
   path: ^1.6.0
   yaml: ^3.0.0
   mdc_web: ^0.6.0
-  sass_builder: ^2.1.0
+  # ^2.1.5 is no longer compatible with our overridden sass version
```

Comment:
Please add an issue to track this. I'm guessing we need to rectify this hard version pin at some point.